komorebi-bar/src/config.rs

```diff
@@ -164,3 +164,10 @@ impl From<KomorebiTheme> for KomobarTheme {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, JsonSchema)]
+pub enum LabelPrefix {
+    Icon,
```

Comment:
Everything looks good, let's just add some basic doc strings (`/// Show an icon`, `/// Show text`, `/// Show an icon and text`) here so they get picked up with the `JsonSchema` derive and can be shown to people using autocompletions in their editors

Reply:
Nice! I will do that. 

Also I was thinking about adding a 4th option called `None` for the real minimalists out there.

Reply:
Would you have any comments on the default values on the widgets?

Battery: Icon
Cpu: IconAndText
Date: Icon
Memory: IconAndText
Network: Icon
Storage: IconAndText
Time: Icon

 ---

I would also like to get your opinion on the text options:

Battery: `BAT`
Cpu: `CPU`
Date: `DATE`
Memory: `RAM`
Network: 
-`UP`, `DOWN`, 
-∑ => `\u{2211}UP`, `\u{2211}DOWM` / `∑UP`, `∑DOWM` or `TUP`, `TDOWM`
Storage: (just the drive letter?)
Time: `TIME`